README.adoc

```diff
@@ -198,17 +172,25 @@ include::{common-includes}/mvnbuild.adoc[]
 
 When the server is running, point your browser to the
 `http://localhost:9080/inventory/systems/localhost` URL.
-You receive a result in JSON format with the system properties of your local JVM. Next, point your
-browser to the `http://localhost:9080/system/properties` URL, which retrieves the information for a
-specific host. Notice that the results from the two URLs are identical.
+You receive the system properties of your local JVM from the `inventory` service. Next, point your
+browser to the the `system` service URL, which is `http://localhost:9080/system/properties`,
+to retrieve the system properties for the specific localhost.
+Notice that the results from the two URLs are identical because `inventory` service gets its results from
+calling the `system` service.
 
-You can test the fault tolerance mechanism of your application by dynamically changing
+You can test the fault tolerance mechanism of your microservices by dynamically changing
 the `io_openliberty_guides_system_inMaintenance` property value to `true` in the
-`resources/CustomConfigSource.json` file. After saving the file, go back to your browser and
+`resources/CustomConfigSource.json` file. This turns the `system` service down in maintenance.
```

Comment:
We can't use "this" as a standalone noun.
I'd add the second sentence onto the first, such as, "...file, which turns..."

Do we need to format `down` as code and state what kind of variable it is, such as "...service to the `down` position in..."?

Reply:
Fixed. I was going combine the two, but it gets too long. All good.